specification/overview.md

```diff
@@ -26,21 +26,58 @@ This specification is designed to be used by three major groups:
+The goal is not to get to a complete, finished specification in one pass. Releasing incremental iterations of the specification will provide value to practitioners and allow feedback as the specification develops.
 
 ### Working Backwards
 
-The FOCUS group working on the specification aims to work backwards from essential FinOps capabilities that practitioners need to perform to prioritize the dimensions, metrics and the attributes of the billing data that should be defined in the specification to fulfill that capability. Some of the enabled capabilities will be documented in the [Appendix: FinOps Capabilities Enabled by FOCUS (TODO)](#finopscapabilitiesenabledbyfocus).
+* The FOCUS group working on the specification aims to work backwards from essential FinOps capabilities that practitioners need to perform to prioritize the dimensions, metrics and the attributes of the cost and usage data that should be defined in the specification to fulfill that capability.
+* Be FinOps scenario-driven. Define columns that answer scenario questions; don't look for scenarios to fit a column, each column must have a use case.
+* Don't add data just because it can be added.
+
+### Ease of adoption is important
+
+* When defining the specification, consideration should be made to existing data already in the major providers' data sources.
+* As long as it solves the FinOps use case, there should be a preference to align with data that is already present in a majority of the major providers.
+* Strive for Simplicity. However, prioritize Accuracy, Clarity, and Consistency.
+* Columns should have a single purpose, with clear and concise names and values.
+* Data should be human-readable, free from jargon, using simple understandable terms, and be approachable.
+* Naming and terms used should be carefully considered to avoid using terms for which the definition could be confused by the reader. If a term must be used which has either an unclear or multiple definitions, it should be clarified in the [glossary](#glossary).
+* The specification should provide all of the data elements necessary for the FinOps capabilities.
+
+### Provider-Neutral Approach by Default
+
+* While the schema, naming, terminology, and attributes of many providers are reviewed during development, this specification aims to be provider-neutral.
+* Contributors must take care to ensure the specification examines how each decision relates to each of the major cloud providers and SaaS vendors, not favoring any one.
```

Comment:
This feels out of step with the previous section, which implies that the "major providers" get preference of consideration.